library/std/src/lib.rs

```diff
@@ -516,6 +516,9 @@ pub mod task {
 mod sys_common;
 mod sys;
 
+#[unstable(feature = "is_atty", issue = "80937")]
+pub use sys::stdio::{Stderr, Stdin, Stdout};
```

Comment:
None of the items inside `sys` should be exported. This is solely an implementation detail. Instead `is_atty` should be on the `Std*` types inside `std::io`. You could add an `IsAtty` trait to each of the relevant platform modules inside the `os` module and implement it there. This should likely forward to a method on `sys::stdio::Std*` though. The `os` module is where platform specific extensions should be.